### PR TITLE
Added test for disconnected node when reading XMLBIF model from files

### PR DESCRIPTION
**Pull request by lohani2280**
Refers https://github.com/pgmpy/pgmpy/issues/938

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [x] Make sure you are making a pull request against the **dev branch** (left side). Also you should start *your branch* off *our dev*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

#### Changes
Please list the proposed changes in this pull request.
- Added test for disconnected node when reading XMLBIF model from files 
-  Added test for disconnected node when reading and writing BIF model.
   In the case of BIF, had to also change the tests because when
    adding nodes manually to the network, networkx also adds a new parameter weight to each node. 
    Since this doesn't affect us in any way elsewhere, I have modified the tests to expect this parameter 
    in nodes.
    Moreover, in order to remove the inconsistency everytime the BIF model file is written, i have used 
    OrderedDict in get_properties method of BIFWriter class.

💔Thank you!

### CODE REVIEW

File: pgmpy/tests/test_readwrite/test_XMLBIF.py [hunk 1]
```diff
@@ -173,6 +187,8 @@ def test_get_values(self):
                                                 [0.85]]),
                         'hear_bark': np.array([[0.7, 0.01],
                                                [0.3, 0.99]]),
+                        'kid': np.array([[0.3],
+                                        [0.7]]),
```
Comment (ankurankan) [inline pgmpy/tests/test_readwrite/test_XMLBIF.py:191]: @lohani2280 PEP8. The second line should be aligned with the second opening bracket of the last line. In short need one extra space on this line.

File: pgmpy/tests/test_readwrite/test_XMLBIF.py [hunk 2]
```diff
@@ -249,6 +268,8 @@ def test_get_values(self):
...
```
Comment (ankurankan) [inline pgmpy/tests/test_readwrite/test_XMLBIF.py:272]: @lohani2280 Same.

**Review by ankurankan — CHANGES_REQUESTED** [review]
Looks good except for the minor comments I put.

File: pgmpy/tests/test_readwrite/test_XMLBIF.py (hunk 1 above)
Comment (lohani2280) [reply pgmpy/tests/test_readwrite/test_XMLBIF.py:191]: @ankurankan Yeah i'll update this. I have added a new commit just now. Please review that one too.

**Review by ankurankan — APPROVED** [review]

**Pull request merged**